README.md

```diff
@@ -216,6 +217,10 @@ export PLUGIN_ADMIN_SECRET="your-secret-here"  # Required for management API
 # Optional environment variables
 export LOCK_TTL_SECONDS=10              # default: 30, min: 3, max: 30
 export MAX_FEE=1000000                  # default: 1,000,000 stroops
+
+# Fee tracking (optional)
+export FEE_LIMIT=1000000                # Max fee per API key in stroops (disabled if not set)
```

Comment:
Are we reseting usage ever, like per day, or this is total value and creation of a new key is required to continue to use service?